.github/workflows/unit-test.yml

```diff
@@ -26,13 +26,10 @@ jobs:
       - name: Install Go dependencies
         run: go mod download
       
-      - name: Build and Test
+      - name: Build and Testga.
+        working-directory: ${{ github.workspace }}
```

Comment:
why do we need this now?

Reply:
i added `mkdir` call to create `coverage/ut` directory in the ut script. without this `mkdir` failed with permission denied. i will try `CWD` in a bit and see how it goes